apps/monitoring/sbox/00/botkube-values.enc.yaml

```diff
@@ -14,12 +14,12 @@ sops:
         - vault_url: https://dcdcftappssboxkv.vault.azure.net
           name: sops-key
           version: 4dfa9dd4b0444f03bd64e2128e347537
-          created_at: "2025-02-21T17:16:54Z"
-          enc: rh2v95OGGlPsRGeJGLkMlJE9anFVp41Zbp7IFrgF-l7DNoQOLgZyDd40ycpRncdGswfh-PwXQqCChoGeEVJHvtVQ5k8kZob9fOrDHMYsuLX3vUs6aqUcWhIXlkVislAGQxnjTC4ouYX8BE-4YmYRxTo7YTMrYbc_C9lpj-DI89rZMsw39AbhdakgMoy-O7jV9N9vfCgzTtZQUy05LsDhOBzVtqnVUKc4YmCfTmrNafuC-5U9d67y2jYhGbJGQSCVJogEq_X4Y_3-_VMhYPwmbIzgpqAj8slOyYDYZiDLRbq415hru-erVVOz1uOE4ik53QT5E7DNi5DQkf2v-Fdf-Q
+          created_at: "2025-04-10T12:23:22Z"
+          enc: TcDt9YRvrm_40wLVip26tgjFcMYIwSNcubDIYCjCowOxgCiqSOvFpOjF24Yg3mLVOT79jROsULB8AZ5ZAGiExQNaIiyC5SB-_GZDIjlgCJ8P9HuISG236daxNNX3cngyprWsiZjQKdE1SCQya0zSXbXs6OeRsfzu-k_hjtmMoHVzb2U5yWSRPtwsAe6AicuaaKO_i6kEPpiOy2kHVrDqdiIXl3shrBWCa_p1uanT1MyugbPoAYch6cN9N4LhZpC0S7CRSr-w9D1Nzo-LSFvWoE-FwjwzOm-4sVSuO5oMvwl5c0EeWyfDWqPd-8WfHdtukKMQqOOfDtP_4cLdIixAlw
     hc_vault: []
     age: []
-    lastmodified: "2025-02-21T17:16:57Z"
-    mac: ENC[AES256_GCM,data:Q7tS1HatammBVg85nyAkQd6q0lXG5sFaYmRYWluLfVS4bWVGn0K+0xnrwECzNeOGt2ySwoFCQLvM0wNy4A5Bn0cEMdUSERXuUvHhh3JaMBvkQKE38Ma5XTWSP9ae29moLVV+dubhyd/VbD8cegitEpaFefe/HDV/SAXhQCpsz2Q=,iv:vHyFX4n/ZXiiRhIjaBeTjQZ1cyvs3VWDtQzEFC/q+vs=,tag:bg4wLq8IzTRnM1EyMKbLuA==,type:str]
+    lastmodified: "2025-04-10T12:23:27Z"
+    mac: ENC[AES256_GCM,data:tznwOnhnx71OuWJC1zOuRrA4six/QkQv6HBYL/P25/pmXWcPO8Cm6av0jv8DFyUujFSjvtBANXXjbEm2zHMhKFKKn7U1RKdr9aYEhOhVudb7MdWj6O+cgVVkwsrF77oYvlz98GpJu81G0RzMmWGCo5oSyxR5hq7FfH2NszvFlGY=,iv:y3rMy4lbuvzWWGL85Nsh9UZCkhpDHTL6jrRqom4JThk=,tag:rkq01G5yBMFzjato5CCDZg==,type:str]
     pgp: []
     encrypted_regex: ^(data|stringData)$
-    version: 3.9.4
+    version: 3.8.1
```

Comment:
Most likely not an issue - but is this because you've used an older version of sops to encrypt this secret than the last person?

Reply:
Yes think its my local sops version.